docs/Using-Pantheon/RPC-PubSub.md

```diff
@@ -0,0 +1,253 @@
+description: Using RPC Pub/Sub with Pantheon Web Socket API
+<!--- END of page meta data -->
+
+# RPC Pub/Sub
+
+## Introduction 
+
+Use the RPC Pub/Sub methods over [WebSockets](../Reference/Using-JSON-RPC-API.md#endpoint-address-and-port) 
+to wait for events instead of polling for them. For example, if a Dapp submits a transaction to the network, 
+you can use the RPC Pub/Sub interface to wait for the transaction confirmation followed by a specified 
+number of new blocks before confirming the transaction to the user.
+
+The RPC Pub/Sub methods are: 
+
+* `eth_subscribe` - call to create a subscription for specific events.
+* `eth_unsubscribe` - call to cancel a subscription.
+* `eth_subscription` - invoked to publish event notifications to your client. 
+
+!!!info
+    Notifications are published by `eth_subscription`; you do not need to call `eth_subscription`.
+
+!!!important
+    Unlike other [JSON RPC-API methods](../Reference/JSON-RPC-API-Methods.md), 
+    the RPC Pub/Sub methods cannot be called over HTTP as well as WebSockets. 
+
+### Subscription ID
+
+`eth_subscribe` returns a subscription ID for each subscription created. Notifications include the subscription ID. 
+
+
+!!!example
+    For example, to create a synchonizing subscription
+  
+    ```json
+    {"id": 1, "method": "eth_subscribe", "params": ["syncing"]}
+    ```
+    The result includes the subscription ID of `"0x1"`: 
+    ```json
+    {"jsonrpc":"2.0","id":1,"result":"0x1"}
+    ```
+    The notifications also include the subscription ID of `"0x1"`: 
+    ```json
+    {"jsonrpc":"2.0","method":"eth_subscription","params":{"subscription":"0x1","result":{"startingBlock":"0x0","currentBlock":"0x50","highestBlock":"0x343c19"}}}
+    ```
+
+### Notifications when Synchronizing
+
+Subscribing to some events (for example, logs) can cause a flood of notifications while the node is synchronizing.
+
+Notifications are stored in an internal buffer to be sent to the client. 
+If the client falls behind and the number of buffered notifications reaches the limit of ==10,000==, the connection is closed.
```

Comment:
Same comment as earlier - the yellow highlight really draws my sight to this first and I'm not sure that's actually helpful.

Reply:
Feel free to propose a way to format these datas, given that we now have a lot of means to do it and not just back-ticks that we should keep only for code.